src/test/unit_tests/test_nc_nsfs_health.js

```diff
@@ -93,22 +93,20 @@ mocha.describe('nsfs nc health', function() {
     });
 
     mocha.describe('health check', async function() {
-        const acount_name = 'account1';
-        const bucket_name = 'bucket1';
         const new_buckets_path = `${root_path}new_buckets_path_user1/`;
-        const account1 = {
-            _id: '1',
-            name: acount_name,
-            nsfs_account_config: {
-                uid: process.getuid(),
-                gid: process.getgid(),
-                new_buckets_path: new_buckets_path
-            }
+        const account1_options = {
+            name: 'account1',
+            uid: process.getuid(),
+            gid: process.getgid(),
+            new_buckets_path: new_buckets_path
+        };
+        const bucket1_options = {
+            name: 'bucket1',
+            path: new_buckets_path + '/bucket1'
         };
-        const bucket1 = { name: bucket_name, path: new_buckets_path + '/bucket1' };
-        const account_inaccessible = { _id: '2', name: 'account_inaccessible', nsfs_account_config: { uid: 999, gid: 999, new_buckets_path: bucket_storage_path } };
-        const account_inaccessible_dn = { _id: '3', name: 'account_inaccessible_dn', nsfs_account_config: { distinguished_name: 'inaccessible_dn', new_buckets_path: bucket_storage_path } };
-        const invalid_account_dn = { _id: '4', name: 'invalid_account_dn', nsfs_account_config: { distinguished_name: 'invalid_account_dn', new_buckets_path: bucket_storage_path } };
+        const account_inaccessible = { name: 'account_inaccessible', uid: 999, gid: 999, new_buckets_path: bucket_storage_path };
```

Comment:
like the variables above
```suggestion
        const account_inaccessible_options = { name: 'account_inaccessible', uid: 999, gid: 999, new_buckets_path: bucket_storage_path };
```